src/components/Messages/ConversationListItem.tsx

```diff
@@ -39,11 +39,15 @@ const ConversationListItem = ({
   onClick,
   selected,
 }: ConversationListItemProps) => {
+  const handleAvatarClick = (e: React.MouseEvent) => {
+    e.stopPropagation();
+  };
+
```

Comment:
I think we can inline this since we're not doing that much and it's not reused

Reply:
Alright so I've updated it to look like this instead

```
<ListItemAvatar
          onClick={(e: React.MouseEvent) => e.stopPropagation()}
          sx={styles.avatar}
        >
          <UserAvatar userId={conversation.otherUserId} />
        </ListItemAvatar>
        ```

Reply:
Nice!